babel.cfg

```diff
@@ -1,5 +1,5 @@
 [python: **.py]
 [jinja2: **.html]
 encoding = utf-8
-extensions=jinja2.ext.autoescape,jinja2.ext.with_
+extensions=jinja2.ext.autoescape,jinja2.ext.with_,warehouse.utils.html:ClientSideIncludeExtension
```

Comment:
I didn't check out your branch but I came up with exactly the same solution so I guess it works. But I think you should do https://github.com/pypa/warehouse/pull/6852/files#r335553211 before merging.